From: Peter Zijlstra <pzijlstr@redhat.com>
Subject: [PATCH RHEL5] fix stex_intr signature
Date: Wed, 06 Dec 2006 12:24:34 +0100
Bugzilla: 219370
Message-Id: <1165404274.19321.10.camel@taijtu>
Changelog: scsi: fix stex_intr signature



While building a RHEL-5 kernel I noticed:
  drivers/scsi/stex.c:1225: warning: passing argument 2 of ‘request_irq’
                                     from incompatible pointer type

Which made me suspect a post pt_regs cleanup driver got inserted in our
tree. As indeed turned out to be the case.

---
 drivers/scsi/stex.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.18.noarch/drivers/scsi/stex.c
===================================================================
--- linux-2.6.18.noarch.orig/drivers/scsi/stex.c
+++ linux-2.6.18.noarch/drivers/scsi/stex.c
@@ -871,7 +871,7 @@ update_status:
 	readl(base + IMR1); /* flush */
 }
 
-static irqreturn_t stex_intr(int irq, void *__hba)
+static irqreturn_t stex_intr(int irq, void *__hba, struct pt_regs *regs)
 {
 	struct st_hba *hba = __hba;
 	void __iomem *base = hba->mmio_base;

